scripts/devenv.sh

```diff
@@ -9,6 +9,7 @@ docker run -it \
 	--privileged \
 	-v /var/run/docker.sock:/var/run/docker.sock \
 	-v `pwd`:/gopath/src/github.com/Azure/acs-engine \
+	-v ~/.azure:/root/.azure \
```

Comment:
This is the devenv.sh in acs-engine.

The line you added implies that 
1. ~/.azure file exist
2. user need az cli

For users want to run this script first, and then login to az cli. It will break, because ~/.azure never exists.